config/locales/devise.two_factor_authentication.en.yml

```diff
@@ -3,7 +3,7 @@ en:
     errors:
       messages:
         select_two_factor: You must select a two factor authenticaton preference.
-        user_not_authenticated: Access denied. Please confirm your one-time passcode to continue.
+        user_not_authenticated: Please enter your one-time password to continue.
```

Comment:
Do we need this flash message?  The page itself is going to already contain these instructions.